ansible/local_testing/Vagrantfile

```
@@ -2,19 +2,20 @@
 # vi: set ft=ruby :
 
 Vagrant.configure("2") do |config|
-    config.vm.box = "bento/debian-11"
-    config.vm.box_version = "202212.11.0"
+    config.vm.box = "bento/debian-12.5"
+    config.vm.box_version = "202404.23.0"
     config.vm.provision "shell", inline: <<-SHELL
-        apt-get update
+        sudo apt-get update
         apt-get install -y python3 openssh-server
+        apt-get install -y python3 ansible
```

Let's use a single command:
```suggestion
        apt-get install -y python3 openssh-server ansible
```
Also I believe in the repository we currently install Ansible via poetry. Do you believe it's possible to also do that here?

---

I think I'd still be in favour to use our regular Poetry version, because if we implement changes such as #264 and also in general the playbook is somewhat coupled to the Ansible version. I don't expect us to break compatibility much but I think to just prevent the possibility of breaking it for the Vagrant setup in the first place we should install it from Poetry. (It's also available as a Debian package)